vizro-core/src/vizro/models/_base.py

```diff
@@ -100,7 +100,7 @@ def _dict_to_python(object: Any) -> str:
             fields = ", ".join(f"{field_name}={_dict_to_python(value)}" for field_name, value in object.items())
             return f"vm.{__vizro_model__}({fields})"
     elif isinstance(object, dict):
-        fields = ", ".join(f"{field_name}={_dict_to_python(value)}" for field_name, value in object.items())
+        fields = ", ".join(f"{field_name}:{_dict_to_python(value)}" for field_name, value in object.items())
```

Comment:
Should we add a space after the colon `":"` -> `": "` to make it look better?
```suggestion
        fields = ", ".join(f"{field_name}: {_dict_to_python(value)}" for field_name, value in object.items())
```

Reply:
Ok I added it, but it would anyway be changed by linting I suppose?